static/app/components/group/groupSummary.tsx

```diff
@@ -0,0 +1,141 @@
+import marked from 'sentry/utils/marked';
+import {type ApiQueryKey, useApiQuery} from 'sentry/utils/queryClient';
+
+interface GroupSummaryProps {
+  group: Group;
+}
+
+interface GroupSummaryData {
+  group_id: string;
+  impact: string;
+  summary: string;
+}
+
+const makeGroupSummaryQueryKey = (groupId: string): ApiQueryKey => [
+  `/issues/${groupId}/summarize/`,
+];
+
+const useGroupSummary = (groupId: string) => {
+  return useApiQuery<GroupSummaryData>(makeGroupSummaryQueryKey(groupId), {
+    staleTime: Infinity, // Cache the result indefinitely as it's unlikely to change if it's already computed
+  });
+};
+
+export function GroupSummary({group}: GroupSummaryProps) {
+  const {data, isLoading, error} = useGroupSummary(group.id);
+
+  return (
+    <SidebarSection.Wrap>
+      <Wrapper>
+        <StyledTitleRow>
+          <StyledTitle>
+            <span>{t('Issue Summary')}</span>
+            <StyledFeatureBadge
+              type="internal"
+              title={tct(
+                'This feature is currently only testing internally. Please let us know your feedback at [channel:#proj-issue-summary].',
+                {
+                  channel: <a href="https://sentry.slack.com/archives/C07GPS55GUC" />,
+                }
+              )}
+              tooltipProps={{isHoverable: true}}
+            />
+          </StyledTitle>
+          {isLoading && <StyledLoadingIndicator size={16} mini />}
+        </StyledTitleRow>
+        <StyledContent>
+          {error ? <div>{t('Error loading summary')}</div> : null}
+          {data && (
+            <Content>
+              <SummaryContent
+                dangerouslySetInnerHTML={{
+                  __html: marked(data.summary),
+                }}
+              />
+              <ImpactContent>
+                <StyledTitle>{t('Potential Impact')}</StyledTitle>
+                <SummaryContent
+                  dangerouslySetInnerHTML={{
+                    __html: marked(data.impact),
+                  }}
+                />
+              </ImpactContent>
+            </Content>
+          )}
+        </StyledContent>
+      </Wrapper>
+    </SidebarSection.Wrap>
+  );
+}
+
+const Wrapper = styled(Panel)`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 0;
+  background: linear-gradient(
+    269.35deg,
+    ${p => p.theme.backgroundTertiary} 0.32%,
+    rgba(245, 243, 247, 0) 99.69%
+  );
+  padding: ${space(1.5)} ${space(2)};
+`;
+
+const StyledTitleRow = styled('div')`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const StyledTitle = styled('span')`
+  margin: 0;
+  color: ${p => p.theme.text};
+  font-size: ${p => p.theme.fontSizeMedium};
+  font-weight: 600;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+`;
+
+const StyledFeatureBadge = styled(FeatureBadge)`
+  margin-top: -1px;
+`;
+
+const StyledContent = styled(SidebarSection.Content)`
+  margin: 0;
+`;
```

Comment:
probably doesn't make sense to use some of these elements if all they provide is margin